src/convexhull.jl

```diff
@@ -0,0 +1,32 @@
+function UpperConvexHull(mySet::PointSet) 
```

Comment:
Function names in Julia are lowercase, can you please rename to `upperhull` and rename the argument to `pset`?